### DISCUSSION TITLE
Compiling and running ILSpy on OS X

### DISCUSSION THREAD

**Discussion by shouldice** [icsharpcode/ILSpy#2091] [General] [answered]
Hello, I'm trying to use ILSpy to decompile my Unity game and test some assembly obfuscation.

I've followed the Unix / Mac instructions in the [readme](https://github.com/icsharpcode/ILSpy/blob/master/README.md), but don't know what to do next. `dotnet build Frontends.sln ` reports a successful build, but then what?

Thanks!

**Comment by christophwille** [comment, marked as answer]
This will give you the command line tool https://www.nuget.org/packages/ilspycmd/

If you want a UI, you might want to check out https://github.com/icsharpcode/AvaloniaILSpy/

**Reply by shouldice** [reply]
Thank you!
